Doc/library/datetime.rst

```diff
@@ -2496,11 +2496,7 @@ Notes:
 (1)
    Because the format depends on the current locale, care should be taken when
    making assumptions about the output value. Field orderings will vary (for
-   example, "month/day/year" versus "day/month/year"), and the output may
-   contain Unicode characters encoded using the locale's default encoding (for
-   example, if the current locale is ``ja_JP``, the default encoding could be
-   any one of ``eucJP``, ``SJIS``, or ``utf-8``; use :meth:`locale.getlocale`
-   to determine the current locale's encoding).
+   example, "month/day/year" versus "day/month/year").
```

Comment:
```suggestion
   example, "month/day/year" versus "day/month/year"), and the output may
   contain non-ASCII characters.
```

I think this change is a net positive, but I'm wondering if we might want to give people a head's up that they shouldn't assume that the output is ASCII, even though `str` is now a Unicode object? Maybe the world is at a point where we don't have to point this out, but maybe it won't hurt?